ingest/src/ingest/__init__.py

```
@@ -0,0 +1 @@
+"""EgoGraphのためのデータ取り込み（Ingestion）モジュール。"""
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)